src/constants/typography/primaryTypography.ts

```diff
@@ -0,0 +1,18 @@
+const fontSize = {
```

Comment:
These values are duplicated from `src/constants/typography/index` because I don't think we should be wrapping them in `throwOnUndefinedProperty`. The value-add is less certain the more we have converted over to TS.